src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -2095,7 +2098,7 @@ private MemberDeclarationSyntax ParseMemberDeclarationOrStatementCore(SyntaxKind
                     _termState |= TerminatorState.IsPossibleStatementStartOrStop; // partial statements can abort if a new statement starts
 
                     // Any expression is allowed, not just expression statements:
-                    var statement = this.TryParseStatementNoDeclaration(attributes, allowAnyExpression: true);
+                    var statement = this.TryParseStatementNoDeclaration((attributes, allowAnyExpression: true));
```

Comment:
This looks absolutely ridiculous but it makes the difference between getting inlined or not on netframework x64.

Reply:
yowza.  perhaps file bug on them?